Directory.Build.props

```diff
@@ -10,7 +10,7 @@
       <WarningLevel>3</WarningLevel>
 
       <VersionPrefix>0.8.0</VersionPrefix>
-      <VersionSuffix>alpha-3</VersionSuffix>
+      <VersionSuffix>alpha-4</VersionSuffix>
```

Comment:
or should it be RC1 ?

Reply:
I think if we don't release we din't need to change it.